src/services/createCommunity.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
```

Comment:
Rename service to community.js
so we can also add other functions related to community